openfisca_us/variables/finance/tax/outputs.py

```diff
@@ -3,44 +3,73 @@
 from openfisca_us.tools.general import *
 
 
-class standard_deduction(Variable):
+class basic_standard_deduction(Variable):
     value_type = float
     entity = TaxUnit
-    label = u"Standard deduction, including for dependents, aged and blind"
+    label = "Basic standard deduction"
     definition_period = YEAR
 
-    def formula(taxunit, period, parameters):
-        # calculate basic standard deduction
-
+    def formula(tax_unit, period, parameters):
         STD = parameters(period).tax.deductions.standard
-        MARS = taxunit("MARS", period)
-        MIDR = taxunit("MIDR", period)
-        MARSType = MARS.possible_values
+        MARS = tax_unit("MARS", period)
+        MIDR = tax_unit("MIDR", period)
         c15100_if_DSI = max_(
-            350 + taxunit("earned", period), STD.amount.dependent
+            350 + tax_unit("earned", period), STD.amount.dependent
         )
         basic_if_DSI = min_(STD.amount.filer[MARS], c15100_if_DSI)
         basic_if_not_DSI = where(MIDR, 0, STD.amount.filer[MARS])
         basic_stded = where(
-            taxunit("DSI", period), basic_if_DSI, basic_if_not_DSI
+            tax_unit("DSI", period), basic_if_DSI, basic_if_not_DSI
         )
+        return basic_stded
 
-        # calculate extra standard deduction for aged and blind
+
+class aged_blind_standard_deduction(Variable):
```

Comment:
Current name could be read as the full stded for them
```suggestion
class aged_blind_extra_standard_deduction(Variable):
```